src/regression_1000.c

```diff
@@ -1395,7 +1395,11 @@ ZTEST(regression_1000, test_1020)
 		ADBG_Assert(&c);
```

Comment:
perhaps this `ADBG_Assert(&c);` should no be there, similar to other tests with `pta_invoke_tests_ta_uuid`

Reply:
Great observation. thanks. Fixed